utils/lib/parse_pcf.py

```diff
@@ -2,6 +2,7 @@
 import re
```

Comment:
Please add a docstring here describing the supported pcf commands (set_io and set_clk)

Reply:
Added